src/stylesheets/blocks/menu/menu__items.pcss

```diff
@@ -11,4 +11,5 @@
   overflow-x: hidden;
 
   background: var(--menu-back-color);
+  transform: scale(1);
```

Comment:
Fix for menu height at safari

![image](https://user-images.githubusercontent.com/670959/109657686-d2ec7000-7b76-11eb-83f5-050050c2f485.png)

Reply:
Please add a comment in the code about this hack. In the future, we will understand what makes this property.

Reply:
I've remove that hack.
More details here https://github.com/uploadcare/uploadcare-widget/pull/813/#discussion_r586266891